src/src_cpp/src_sequential/launcher.cpp

```diff
@@ -84,6 +84,9 @@ void sequential_launch(BendersOptions const & options) {
 	build_input(options, input);
 
 	JsonWriter jsonWriter_l;
+	jsonWriter_l.write_failure();
+	jsonWriter_l.dump(options.OUTPUTROOT + PATH_SEPARATOR + "out.json");
+
```

Comment:
why dump twice ?

Reply:
Is the file overwritten on the second dump ?

Reply:
It is.